Kitodo-DataAccess/src/test/java/org/kitodo/dataaccess/LangStringTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+import org.kitodo.dataaccess.LangString;
+import org.kitodo.dataaccess.Node;
+import org.kitodo.dataaccess.RDF;
+import org.kitodo.dataaccess.Storage;
+
+public class LangStringTest {
+
+    private static final String XML_LANG = "http://www.w3.org/XML/1998/namespace#lang";
+
+    @Test
+    public void testEqualsObject() {
+        for (Storage storage : TestConfig.STORAGES_TO_TEST_AGAINST) {
+            LangString one = storage.createLangString("Hoc est corpus meum.", "la");
+            LangString another = storage.createLangString("Hoc est corpus meum.", "la");
+
+            assertThat(one, is(equalTo(another)));
+        }
+    }
+
+    @Test
+    public void testHashCode() {
+        for (Storage storage : TestConfig.STORAGES_TO_TEST_AGAINST) {
+            LangString one = storage.createLangString("Hoc est corpus meum.", "la");
+            LangString other = storage.createLangString("Hoc est corpus meum.", "la");
+
+            assertThat(other.hashCode(), is(equalTo(one.hashCode())));
+        }
+    }
+
+    @Test
+    public void testLangStringCanBeCreatedWithBothTextAndLanguageGiven() {
+        for (Storage storage : TestConfig.STORAGES_TO_TEST_AGAINST) {
+            storage.createLangString("In vino veritas est.", "la");
```

Comment:
What this kind of test? You are not testing anything.

Reply:
No, I am testing that the method `createLangString()` does **not** throw an exception when both parameters are in their range. Testing also means to test the working situation, not only the error cases. Assume the the parameter check for the second parameter would be wrong, like `if(language == null || !language.isEmpty()) throw new IllegalArgumentException("Language must not be null or empty");` then this function would throw an exception and the test would fail. (Correctly, language.isEmpty() must not be negated here.)

Reply:
But this method can't throw any exception (except RuntimeException) as its declaration doesn't contain any.